### ISSUE TITLE
Broken box when using a certain combination of size and padding

### ISSUE THREAD

**Issue by DannyBen** [piotrmurach/tty-box#2]
### Describe the problem

There seems to be a problem with a certain combinations of width, content length and padding.

### Steps to reproduce the problem

Using the very first code from the README, only with `padding: 1` instead of 3:

```ruby
require 'tty-box'

box = TTY::Box.frame(
  width: 30,
  height: 10,
  align: :center,
  padding: 1
) do
  "Drawing a box in terminal emulator"
end

puts box
```

### Actual behavior

The script produces this output:

```
$ ruby test.rb
┌────────────────────────────┐
│                            │
│ Drawing a box in terminal  │
│ emulator │
│          emulator          │
│                            │
│                            │
│                            │
│                            │
└────────────────────────────┘
```

Provided also as a screenshot:

![image](https://user-images.githubusercontent.com/2405099/45265451-a8147b80-b453-11e8-9fb6-cf127b090f1d.png)

### Describe your environment

* OS version: Ubuntu 16
* Ruby version:  2.5.0p0 (2017-12-25 revision 61468) [x86_64-linux]
* TTY::Box version: 0.2.0

**Comment by piotrmurach** [comment]
Thank you for reporting this! Do you have some time to investigate the cause?

**Comment by piotrmurach** [comment]
Released `v0.2.1` with your fix! Thanks again! ❤️

**Comment by DannyBen** [comment]
With pleasure.
